pkg/plugins/resources/updateclihttp/spec.go

```diff
@@ -0,0 +1,32 @@
+package updateclihttp
+
+// Spec defines a specification for a "http" resource
+// parsed from an updatecli manifest file
+type Spec struct {
+	// [S][C] Specifies the URL to use for the HTTP request of this resource
```

Comment:
You may have notice but I started using multiline comment `/* */` so I can provide more information to the generated jsonschema which ultimately benefit to website and IDE integration

You have an example here 
-> https://github.com/updatecli/updatecli/blob/8490900bab55b2a7c153a5f94c53faec6aded5f3/pkg/plugins/resources/yaml/main.go#L18

Which render like 
-> https://www.updatecli.io/docs/plugins/resource/yaml/

It's a personnal taste but I like multiline comments as I find them easier read 

Remark:

The only thing to keep in mind is that I noticed that sometimes go fmt reformat comment using tab instead of space which doesn't render the same in IDE

Reply:
By the way, not all plugins use the multi line approach, I usually migrate them one by one when I work on them

Reply:
Treated in https://github.com/updatecli/updatecli/pull/1773/commits/494ac44c90ba03bc98496c37b724efa2a894ff8c